.travis.yml

```diff
@@ -47,10 +45,11 @@ script:
     - printf $(git rev-parse HEAD) > .application-version
     - make travis
     - export TAG=`if [ "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]; then echo "latest"; else echo branch-$(echo $TRAVIS_PULL_REQUEST_BRANCH | sed s:/:-:g); fi`
-    - docker tag eu.gcr.io/census-eq-ci/eq-survey-runner:v3 eu.gcr.io/census-eq-ci/eq-survey-runner:"$TAG"
-    - echo "$DOCKER_GCR_PASSWORD" | base64 --decode | docker login -u "$DOCKER_GCR_USERNAME" --password-stdin https://eu.gcr.io
+    - docker build -t onsdigital/eq-questionnaire-runner:$TAG -f Dockerfile .
```

Comment:
This was previously changed to just tag the previously build image rather than build it again.